Week-01/Hayman-homework.txt

```diff
@@ -0,0 +1 @@
+First commit 
```

Comment:
Nice job, @kirkukcenter! This file will be where you write all homework info as you're doing week 1 homework. Whenever it says to write something in the `homework.txt` file, you can write it here, in `Hayman-homework.txt`.

However, it looks like you didn't create a `Hayman-homework.txt` in the `Week-02` folder. There is some homework for week 2 as well, and you'll be writing the homework for week 2 in that file, so you still need to create that one as well.